src/App.tsx

```diff
@@ -17,6 +18,7 @@ const AppRouter = () => (
     <Switch>
       <Route path="/raids" exact component={AllRaids} />
       <Route path="/raids/:raidId" exact component={ViewRaid} />
+      <Route path="/homepage" exact component={HomePage} />
```

Comment:
Why not just work on the actual home `/` route? Or was this only until it was ready

Reply:
I would prefer the `/` be used as well